battleship.py

```diff
@@ -25,7 +25,16 @@
 Returns: None
 '''
 def makeModel(data):
-    return
+    data["no.of rows"]=10
+    data["no.of cols"]=10
+    data["board size"]=500

```

Comment:
instead of space use upper case character or underscore as a separator

Reply:
changed mam